### DISCUSSION TITLE
Is this normal for Trakt import?

### DISCUSSION THREAD

**Discussion by JPDVM2014** [IgnisDa/ryot#689] [General]
I imported my Trakt history, and it seems to have worked; however, no data loads for shows/movies until I click into them. So, my summary will never be correct unless I go through and click into each item.

![Screenshot_20240311_110830_Mull.jpg](https://github.com/IgnisDa/ryot/assets/18446490/5a84407d-31b6-428d-a386-933766e78f9e)

Movie and shows just show as gray boxes with letters until I click into it.
![Screenshot_20240311_111638_Mull_1.jpg](https://github.com/IgnisDa/ryot/assets/18446490/874ec030-bb4a-4d1d-a588-415083f89fc3)

I've run the update metadata and summary job a few times with no change.

EDIT: I realize this probably should have went into Q&A, but I can't see a way to change it.

**Comment by IgnisDa** [comment]
Please create an issue for this. What I can do is the update metadata job to force update all media items.
